syncer/heartbeat.go

```diff
@@ -196,8 +196,8 @@ func (h *Heartbeat) TryUpdateTaskTs(taskName, schema, table string, data [][]int
 		h.logger.Warn("invalid data server_id for heartbeat", zap.Reflect("server ID", latest[1]))
 		return
 	}
-	if int(serverID) != h.cfg.serverID {
-		h.logger.Debug("ignore mismatched server_id for heartbeat", zap.Int32("obtained server ID", serverID), zap.Int("excepted server ID", h.cfg.serverID))
+	if uint32(serverID) != h.cfg.serverID {
```

Comment:
can we convert to `uint32` in L194?

Reply:
I do this at the first, but It will transform failed